projects/bp-gallery/src/components/common/picture-gallery/PictureTimeline.tsx

```diff
@@ -0,0 +1,107 @@
+import { ArrowDropDown } from '@mui/icons-material';
+import { debounce } from 'lodash';
+import { Dispatch, SetStateAction, useCallback, useEffect, useMemo, useRef } from 'react';
+import { useAnimate } from '../../../hooks/animate.hook';
+
+export const enum TimeStepType {
+  DECADE = 'decade',
+  YEAR = 'year',
+  MONTH = 'month',
+  DAY = 'day',
+}
+
+const TEXT_OVERFLOW_WIDTH = 20;
+
+const PictureTimeline = ({
+  start,
+  end,
+  type,
```

Comment:
`type` currently does nothing. Do you intend to leave it in for future expansion? If so, maybe add a comment that says so, otherwise, I'd remove it and the enum above.

Reply:
Yes, it is for future expansion. But I decided, that this non final version, already provides value.